src/lib/signature_lib/keypair.mli

```diff
@@ -5,13 +5,13 @@ module Stable : sig
     type t =
       { public_key: Public_key.Stable.V1.t
       ; private_key: Private_key.Stable.V1.t sexp_opaque }
-    [@@deriving sexp, bin_io]
```

Comment:
How come you didn't use the versioning ppx here?

Reply:
We don't yet have `%%versioned` for signatures.

Reply:
oh, I thought this was an .ml file. Sorry, my bad